1_portfolio_check_initialisation.R

```diff
@@ -30,26 +30,16 @@ source("0_portfolio_input_check_functions.R")
 source("0_global_functions.R")
 source("0_sda_approach.R")
 
-
-# Set Project Settings
-
-# within the "project_setting.yml" config file, set the project_name, the twodii_internal switch,
-# and the external data locations, if necessary.
-# the project_name will determine the name of the folder that is created for this project
-# Set twodii_internal to TRUE to run the analysis on an internal 2dii laptop
-# This setting uses the dropbox connection for data import
-# Set twodii_internal to FALSE, tu use external data locations
-# Specify these data locations in the config file "project_settings.yml" in the repo
-
-cfg <- config::get(file = "project_settings.yml")
```

Comment:
when we merge this with master, we definitely need to go for the config version (i.e. master version)